### ISSUE TITLE
some question about simple_test_sg_bboxes

### ISSUE THREAD

**Issue by flyfaerss** [Jingkang50/OpenPSG#52]
In simple_test_sg_bboxes function, setting the predicated dist all to one-hot is confusing, how do I get the predicted logits from the pre-trained model？ And how can I pre-train the model?

**Comment by king159** [comment]
If you want to pre-train the model, you may refer to the code of psgformer_head.py instead test.py.
